cms/djangoapps/contentstore/views/course.py

```diff
@@ -1359,6 +1360,15 @@ def advanced_settings_handler(request, course_key_string):
                         # now update mongo
                         modulestore().update_item(course_module, request.user.id)
 
+                        # Send other course settings update to OEE plugin only on valid POST.
+                        other_course_settings = updated_data.get('other_course_settings')
+
+                        run_extension_point(
+                            'OEE_UPDATE_COURSE_SETTINGS',
+                            course_key=course_key,
+                            other_course_settings=other_course_settings.get('value') if other_course_settings else {},
```

Comment:
I think this will show a line too long. @ivanvgh

Reply:
No it won't, there are 3 spaces left to be considered a too long line.